src/components/App.tsx

```diff
@@ -762,6 +764,27 @@ function App() {
     if (messages) navigator.clipboard.writeText(messages);
   };
 
+  /*//////////////////////////////////////////////////////////////
+                         RENAME NODE LOGIC
+  //////////////////////////////////////////////////////////////*/
+
+  const showRenameInput = () => {
+    takeSnapshot();
+
+    const selectedNode = nodes.find((node) => node.selected);
+    const nodeId = selectedNode?.id ?? selectedNodeId;
+
+    if (nodeId) {
+      setNodes((nodes) =>
+        modifyFluxNode(nodes, {
+          id: nodeId,
+          type: FluxNodeType.LabelUpdater,
+          draggable: false,
```

Comment:
im ok with label updater nodes being draggable for the sake of eliminating complexity

Reply:
ahhh ok — but only `command + {arrow}` right? i think thats ok? even if you draggable off u just end up disabling command arrow from doing anything at all which isnt much better